tests/Unit/NumericalAlgorithms/LinearOperators/Test_Divergence.cpp

```diff
@@ -269,22 +268,16 @@ SPECTRE_TEST_CASE("Unit.Numerical.LinearOperators.Divergence.ComputeItem",
   const Mesh<3> mesh_3d{{{n0, n1, n2}},
                         Spectral::Basis::Legendre,
                         Spectral::Quadrature::GaussLobatto};
-  for (size_t a = 0; a < 5; ++a) {
-    std::array<std::unique_ptr<MathFunction<1>>, 1> functions_1d{
-        {std::make_unique<MathFunctions::PowX>(a)}};
-    test_divergence_compute_item(mesh_1d, std::move(functions_1d));
-    for (size_t b = 0; b < 4; ++b) {
-      std::array<std::unique_ptr<MathFunction<1>>, 2> functions_2d{
-          {std::make_unique<MathFunctions::PowX>(a),
-           std::make_unique<MathFunctions::PowX>(b)}};
-      test_divergence_compute_item(mesh_2d, std::move(functions_2d));
-      for (size_t c = 0; c < 3; ++c) {
-        std::array<std::unique_ptr<MathFunction<1>>, 3> functions_3d{
-            {std::make_unique<MathFunctions::PowX>(a),
-             std::make_unique<MathFunctions::PowX>(b),
-             std::make_unique<MathFunctions::PowX>(c)}};
-        test_divergence_compute_item(mesh_3d, std::move(functions_3d));
-      }
-    }
-  }
+  std::array<std::unique_ptr<MathFunction<1>>, 1> functions_1d{
+      {std::make_unique<MathFunctions::PowX>(2)}};
+  test_divergence_compute_item(mesh_1d, std::move(functions_1d));
+  std::array<std::unique_ptr<MathFunction<1>>, 2> functions_2d{
+      {std::make_unique<MathFunctions::PowX>(3),
+       std::make_unique<MathFunctions::PowX>(2)}};
+  test_divergence_compute_item(mesh_2d, std::move(functions_2d));
+  std::array<std::unique_ptr<MathFunction<1>>, 3> functions_3d{
+      {std::make_unique<MathFunctions::PowX>(4),
+       std::make_unique<MathFunctions::PowX>(3),
+       std::make_unique<MathFunctions::PowX>(2)}};
+  test_divergence_compute_item(mesh_3d, std::move(functions_3d));
```

Comment:
What's going on here?  This change does not seem tag-related.

Reply:
ah yes, this should be a separate commit; this change and the other you commented on are because I don't see a reason to test the compute item more than once with trivial differences...

Reply:
I just dropped this change for now